components/mpas-ocean/src/shared/mpas_ocn_diagnostics.F

```diff
@@ -78,11 +78,17 @@ module ocn_diagnostics
    real (kind=RKIND), pointer ::  coef_3rd_order
 
    ! Methods for computing thickness at edges for flux calculations
-   integer :: thickEdgeFluxChoice  ! choice of thickness flux type
+   integer :: &
+      thickEdgeFluxChoice, &! choice of thickness flux type
+      thickEdgeDragChoice   ! choice of thickness drag type
    integer, parameter :: &
       thickEdgeFluxUnknown = 0, &! type unknown or unset
       thickEdgeFluxCenter  = 1, &! use mean thickness of cell neighbors
       thickEdgeFluxUpwind  = 2   ! use upwind cell thickness at edge
+   integer, parameter :: &
+      thickEdgeDragUnknown = 0, &! type unknown or unset
```

Comment:
I'm a bit confused about why the `*Unknown` parameters are needed.  Don't we just raise a critical error and stop the simulation right after they get set?  So maybe just raise the critical error and exit without bothering to set a value?

Reply:
```suggestion
```

Reply:
@xylar I think that's a good suggestion. I've removed them with https://github.com/E3SM-Project/E3SM/pull/5475/commits/00847366fe5b7a7afdc1ab8bad59cf9f21385d85